rfcs/45-stageless.md

```diff
@@ -453,33 +452,17 @@ We will use a global code freeze during those steps to avoid merge conflicts, an
 
 ## Rationale and alternatives
 
-### How are system sets any different from labels?
-
-Formally speaking, a system set is a subgraph.
-It's a node in the graph of its parent set(s) (unless it's a root).
-
-We wanted to make it clear that labels are *names* for things and not *tags*.
-You can name systems and system sets.
-
-Today, if you add multiple labels to a system, the expectation is that each label is involved in at least one dependency, so you're actually positioning things, not categorizing them.
-We have made this explicit.
-You position things within sets (and because sets can be "nested", we've added depth).
-
-Although users don't normally add labels and leave them "unconstrained", such a thing does have a direct analogue in this design.
-You can simply include systems in additional sets that are decoupled from the ones you'd normally execute.
-In that case, you'd be using "set" as a simple abstract group instead of a scoped dependency graph.
-
 ### If users can configure system sets, can they control where systems from third-party plugins are added?
 
 To some extent, yes.
 
 Not being able to schedule plugin systems is a source of [significant user pain](https://github.com/bevyengine/bevy/issues/2160).
-We thought plugins exporting system set labels and users deciding where to position them in their schedules makes for a better balance.
+We thought plugins exporting system sets and users deciding where to position them in their schedules makes for a better balance.
 A popular physics plugin [has already implemented something like this](https://github.com/dimforge/bevy_rapier/pull/169) (partly [inspired](https://discord.com/channels/691052431525675048/742569353878437978/970075559679893594) by this effort).
 
 There are three key rules for system configuration:
 
-1. You can always add configuration (for any publically labeled set).
+1. You can always add configuration (for any public, unlocked set).

```

Comment:
What does "unlocked" mean here?

Reply:
As discussed on discord, we've removed the "set locking" feature from the RFC, as it introduced a number of challenging design problems.